src/solver/optim-model-filler/ReadLinearExpressionVisitor.cpp

```diff
@@ -160,7 +163,22 @@ TimeDependentLinearExpression ReadLinearExpressionVisitor::visit(const PortField
 
 TimeDependentLinearExpression ReadLinearExpressionVisitor::visit(const PortFieldSumNode* node)
 {
-    throw std::invalid_argument("ReadLinearExpressionVisitor cannot visit PortFieldSumNodes");
+    std::string port = node->getPortName();
+    std::string field = node->getFieldName();
+
+    TimeDependentLinearExpression to_return(fillContext_, {});
+    for (const auto* c: component_.connections())
```

Comment:
Two issues here, resulting from the fact that a port connection is not fully defined by the components: we need the port IDs. In this code, the following examples will fail:
- if component A is connected to B via their port X, and to C via their port Y
- if component A's port X is connected to B's port Y

Reply:
Done